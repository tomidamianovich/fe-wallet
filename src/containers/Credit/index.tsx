import React from 'react';
import MaintenancePage from "../../components/MaintenancePage";
import { withTranslation, WithTranslation } from 'react-i18next';
import './styles/index.scss';

type Props = WithTranslation;

/*

  Component Name: Credit
  Usages: 
    - Credit Page
    
*/

const Credit: React.FC<Props> = ({t}) => 
  <div className="fe-wallet__credit">
    <MaintenancePage pageName={t('pages.credit')} />
  </div>
;

export default withTranslation()(Credit);