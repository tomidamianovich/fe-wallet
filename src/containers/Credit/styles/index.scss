.fe-wallet__credit {
  display: flex;
  width: 100%;
}