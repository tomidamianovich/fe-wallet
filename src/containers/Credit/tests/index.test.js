import { render } from '@testing-library/react';
import Credit from '../';

test('renders Credit main page', () => {
  const { asFragment } = render(<Credit />);
  expect(asFragment()).toMatchSnapshot();
});
