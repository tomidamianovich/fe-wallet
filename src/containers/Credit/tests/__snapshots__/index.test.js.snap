// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Credit main page 1`] = `
<DocumentFragment>
  <div
    class="fe-wallet__credit"
  >
    <div
      class="fe-wallet__maintenance-page"
    >
      <h1>
        maintenance.title
      </h1>
      <h3>
        maintenance.subtitle.beginningmaintenance.subtitle.ending
      </h3>
    </div>
  </div>
</DocumentFragment>
`;
